use assemble_daemon::error::DaemonError;
use assemble_daemon::Daemon;

fn main() -> Result<(), DaemonError> {
    Ok(())
}
